Code/GraphMol/RingInfo.cpp

```diff
@@ -1,6 +1,5 @@
-// $Id$
 //
-//  Copyright (C) 2004-2008 Greg Landrum and Rational Discovery LLC
+//  Copyright (C) 2004-2016 Greg Landrum and Rational Discovery LLC
```

Comment:
Shouldn’t this be 2019?

Reply:
yeah, that was a leftover from my first attempt to use the library